lua/lint/linters/clj-kondo.lua

```diff
@@ -3,13 +3,17 @@ local severities = {
   warning = vim.lsp.protocol.DiagnosticSeverity.Warning,
 }
 
+local function get_file_name(bufnr)
+  return vim.api.nvim_buf_get_name(bufnr)
+end
```

Comment:
```suggestion
local function get_file_name()
  return vim.api.nvim_buf_get_name(0)
end
```

Currently no arguments are passed to the functions used within `args`.

Reply:
Right, making the change, i was trying to follow https://github.com/mfussenegger/nvim-lint/blob/master/lua/lint/linters/vale.lua#L2